src/components/LiveCursors/Cursors.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from 'react'
+import { Space, SpaceMember } from '@ably-labs/spaces'
+import CursorSvg from './CursorSvg'
+import useCursor from './use-cursor'
+
+// 💡 This component is used to render the cursor of the user
+const YourCursor = ({
+  user,
+  space,
+}: {
+  user: SpaceMember | undefined
+  space: Space | undefined
+}) => {
+  const [cursorPosition, setCursorPosition] = useState<{
+    [key: string]: number
+  }>({ left: 100, top: 100 })
+
+  const handleSelfCursorMove = useCursor(setCursorPosition, space)
+
+  return (
+    <div
+      className="absolute"
+      onMouseMove={(e) => handleSelfCursorMove(e)}
+      style={{
+        top: `${cursorPosition.top + 30}px`,
+        left: `${cursorPosition.left}px`,
+      }}
+    >
+      <CursorSvg
+        startColor={user?.profileData.userColors.split(' ')[2]}
+        endColor={user?.profileData.userColors.split(' ')[3]}
+        id={user?.connectionId}
```

Comment:
We should not render here if a user or his profileData is undefined - as per the other comment, this will just produce broken UI. We should only render when we are confident we have the user (and we need to express that in types as well, by checking that the properties we need exist).

Reply:
as above, now that we're checking if the user is definied we don't need to make the user optional